src/webviews/src/App.tsx

```diff
@@ -73,52 +78,59 @@ function App({}: AppProps) {
 
   return (
     <div className="p-4">
-      <div className="mb-2 p-4">
-        <h1 className="text-2xl font-bold text-vscode-settings-headerForeground py-2">
-          Flat Editor
-        </h1>
-        <p className="mb-2">
-          This is a gui for setting up a Flat Action, which will pull external
-          data and update it using GitHub Actions.
-        </p>
-        <button onClick={handleOpenRaw}>View the raw YAML</button>
-      </div>
-      <Triggers />
-      <Jobs />
-
-      {showErrorState ? (
-        <div className="border-2 border-vscode-editorOverviewRuler-warningForeground bg-vscode-settings-dropdownBackground text-sm p-4 flex items-center">
-          <span className="codicon codicon-warning mr-1 text-vscode-editorOverviewRuler-warningForeground" />
-          <p>
-            Make sure all of your steps have a{' '}
-            <strong>downloaded_filename</strong> specified!
-          </p>
+      <header>
+        <div className="mb-2">
+          <h1 className="text-[26px] leading-[30px] font-medium my-0">
+            Flat Editor
+          </h1>
         </div>
-      ) : (
-        <div className="pt-4 pl-4 pb-6">
-          <p className="text-lg font-bold pb-1">
-            You're all set!{' '}
```

Comment:
I know this text was already here but I was wondering if it's necessary? It sounds like something I'd see at the end of a wizard/stepped flow. It was already there when I opened the editor, so I wasn't sure if I had already completed something or not.

Reply:
Yep, I was also a bit thrown by this text too. I would have expected to be rendered as a result of interacting with the form/options versus being visible from the beginning